Modules/Microsoft365DSC/DSCResources/MSFT_AADEntitlementManagementAccessPackageAssignmentPolicy/MSFT_AADEntitlementManagementAccessPackageAssignmentPolicy.psm1

```diff
@@ -0,0 +1,1572 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.String]
+        $Id,
+
+        [Parameter()]
+        [System.String]
+        $AccessPackageId,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $AccessReviewSettings,
+
+        [Parameter()]
+        [System.Boolean]
+        $CanExtend,
+
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.Int32]
+        $DurationInDays,
+
+        [Parameter()]
+        [system.datetimeoffset]
+        $ExpirationDateTime,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $Questions,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $RequestApprovalSettings,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance]
+        $RequestorSettings,
+
+        [Parameter()]
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $CustomExtensionHandlers,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        [ValidateSet('Absent', 'Present')]
+        $Ensure = $true,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint,
+
+        [Parameter()]
+        [Switch]
+        $ManagedIdentity
+    )
+
+    try
+    {
+        $ConnectionMode = New-M365DSCConnection -Workload 'MicrosoftGraph' `
+            -InboundParameters $PSBoundParameters `
+            -ProfileName 'beta'
+
+        Select-MgProfile 'beta'
+    }
+    catch
+    {
+        Write-Verbose -Message ($_)
```

Comment:
Why a Verbose message here? If creating a connection fails, it doesn't make sense to run the rest of the code.

Reply:
Sorted